package.json

```diff
@@ -24,8 +24,20 @@
     "build-all": "npm run build --workspaces",
     "prepare": "npm run build-all"
   },
-  "main": "./dist/index.js",
-  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
+    },
+    "./ts": {
+      "types": "./dist/ts/index.d.ts",
+      "default": "./dist/ts/index.js"
+    },
+    "./zod": {
+      "types": "./dist/zod/index.d.ts",
+      "default": "./dist/zod/index.js"
```

Comment:
This looks right to me, but figure we can check in with @andrewbranch and @jakebailey.

Reply:
Looks good, but can be simplified to

```
".": "./dist/index.js",
"./ts": "./dist/ts/index.js",
"./zod: "./dist/zod/index.js"
```

TS knows how to find sibling types.

Also, note that anyone accidentally using `--moduleResolution node10` (still a default for some settings) will not be able to access these subpaths. TS will tell them to import from `"typechat/dist/zod"` even though that won’t work in anything modern at runtime. It’s up to you whether you want to use that as an opportunity to get them to update their settings or provide a mitigation like one of these: https://github.com/andrewbranch/example-subpath-exports-ts-compat